.meta/sinks/statsd.toml.erb

```diff
@@ -1,41 +0,0 @@
-[sinks.statsd]
-title = "Statsd"
-noun = "Statsd"
-beta = false
-common = false
-delivery_guarantee = "best_effort"
-<%= render("_partials/descriptions/_statsd.toml") %>
-egress_method = "streaming"
-features = [
```

Comment:
We don't use features anymore?

Reply:
No, I am dropping those, they were for SEO purposes and we don't need them.